monitors/filterWords.js

```diff
@@ -0,0 +1,44 @@
+exports.run = async (client, msg) => {
+  const modlog = msg.guild.settings.modLog;
+  if (msg.deletable) {
+    if (msg.guild.settings.swearWords) {
+      const badWords = msg.guild.settings.swearWords.filter(word => msg.content.toLowerCase().includes(word));
+      if (badWords.length > 0) {
+        const badWordList = [];
+        for (let i = 0; i < badWords.length; i++) {
+          badWordList.push(badWords[i]);
+        }
+        await msg.author.send(`I deleted the message below because you used a word that is not allowed. : **${badWordList.join(", ")}**`);
+        await msg.author.send(msg.content);
+        await msg.delete()
+          .then(client.channels.get(modlog).send(`Deleted message from ${msg.author} that said the \n\n **Naughty Words:** ${badWordList} in the server ${msg.guild} in the channel ${msg.channel}. \n\n **Full Message:** ${msg.content}`))
```

Comment:
`client.channels.get` is a no-no for most cases. Use `msg.guild.channels.get` instead. Modlog channels are always in the same guild the command got executed.